tests/test_3rd_party.py

```diff
@@ -4,13 +4,16 @@
 Tests for compatibility against other Python modules.
 """
 
-import cloudpickle
+import pytest
 
 from hypothesis import given
 
 from .strategies import simple_classes
 
 
+pytest.importorskip("cloudpickle")
```

Comment:
```suggestion
cloudpickle = pytest.importorskip("cloudpickle")
```

Reply:
I'm sorry about the mistake. I should've tested it better.